src/routes/Register.js

```diff
@@ -47,33 +47,38 @@ const Register = ({
   };
   return (
     <>
-      {resModal && (
+    {resModal && (
         <AuthResponseModal resData={resData} setResModal={setResModal} />
       )}
-      <main>
-        <header>
-          <h3>Register</h3>
-        </header>
-        <section>
-          <form id="login" onSubmit={authFormSubmitHandler}>
-            <label>E-mail Address</label>
-            <input
-              id="username"
-              type="email"
-              value={username}
-              onChange={usernameChangeHandler}
-            />
-            <label>Password (must be at least 8 characters)</label>
-            <input
-              id="pasword"
-              type="password"
-              value={password}
-              onChange={passwordChangeHandler}
-            />
-            <button type="submit">Submit</button>
-          </form>
-        </section>
-      </main>
+      <section className="register-page">
```

Comment:
Why change the semantics here? The page should be wrapped in a main, not section. The h3 should be wrapped in a header, and the form should be wrapped in a section.